imageregistry/v1/types.go

```diff
@@ -200,6 +200,11 @@ type ImageRegistryConfigStorageS3 struct {
 	// Optional, defaults based on the Region that is provided.
 	// +optional
 	RegionEndpoint string `json:"regionEndpoint,omitempty"`
+	// The S3 API requires multipart upload chunks to be at least 5MB.
+	// This value should be a number that is larger than 5 * 1024 * 1024.
+	// The default is 10 MB.
+	// +optional
+	Chunksize int `json:"chunksize,omitempty"`
```

Comment:
Why `Chunksize` and not `ChunkSize`?

Reply:
consistency with https://docker-docs.uclv.cu/registry/storage-drivers/s3/